playbooks/azure/openshift-cluster/build_node_image.yml

```diff
@@ -94,6 +94,15 @@
     with_items:
     - { regexp: '^ResourceDisk\.Format=', line: 'ResourceDisk.Format=n' }
 
+  # set waagent to start after docker
+  - name: set waagent systemd file
```

Comment:
Do you need to `systemctl daemon-reload` after this or is it done somewhere down the line?

Reply:
Added below

Reply:
shouldn't be necessary - this is happening at image build time - when the image boots in the cluster it will read the config from disk appropriately